packages/teleport/src/Console/DocumentSsh/DocumentSsh.story.tsx

```diff
@@ -15,11 +15,13 @@ limitations under the License.
 */
 
 import React from 'react';
-import type { Session } from 'teleport/services/session';
+
 import DocumentSsh from './DocumentSsh';
 import { TestLayout } from './../Console.story';
 import ConsoleCtx from './../consoleContext';
 
+import type { Session } from 'teleport/services/session';
```

Comment:
out of order

Reply:
types are last in the config so this is correct.